docs/faq.md

```diff
@@ -256,3 +256,17 @@ Merged objects will always contain all samples in the given project ([see which
 
 If you would like to work with a merged object that only contains a subset of project samples, we recommend downloading the merged object and subsetting it directly to the samples of your choosing.
 See {ref}`Subsetting the Merged Object<getting_started:Subsetting the merged object>` for instructions on how to subset `SingleCellExperiment` and `AnnData` merged objects.
+
+
+## Why did project options change when I appended samples to My Dataset?
+
+It is possible append samples from a dataset you previously created in the past to your current `My Dataset`.
```

Comment:
```suggestion
If you would like to include samples from a dataset you have previously created, you can append these samples to your current `My Dataset`.
```
How about this? This sentence seemed a little off to me.

Reply:
This is much better!!